src/components/DivElement.js

```diff
@@ -1,5 +1,21 @@
 // Import HTMLElement here
+import HTMLElement from './HTMLElement';
 
 // Define class here
+class DivElement extends HTMLElement {
+  constructor(content) {
+    super('div', content);
+  }
+
+  render() {
+    // Instructions unclear.
+    // Sounds like this is what you want:
+    return `<${this.tag}>${this.content}</${this.tag}>`;
+
+    // Easier if it was
+    // return super.render();
+  }
```

Comment:
> This class should inherit the render function in the parent class (do not override it, aka define it in DivElement).

Since `DivElement` inherits from `HTMLElement`, the `render` function doesn't need to be defined. It's inherited from `HTMLElement`. This would be an "override". See: https://javascript.info/class-inheritance#overriding-a-method